src/gui/tray/activitylistmodel.cpp

```diff
@@ -283,6 +299,14 @@ QVariant ActivityListModel::data(const QModelIndex &index, int role) const
         return !data(index, PathRole).toString().isEmpty() && a._objectType == QStringLiteral("files") && _displayActions && a._fileAction != "file_deleted" && a._status != SyncFileItem::FileIgnored;
     case IsCurrentUserFileActivityRole:
         return a._isCurrentUserFileActivity;
+    case ThumbnailRole: {
+        if(a._previews.empty()) {
+            return {};
```

Comment:
see my comment in qml linked to this empty return

Reply:
Got it, fixed in QML